feature/system/tests/system_base_test/g_protocol_test.go

```diff
@@ -58,7 +59,22 @@ func TestGNMIClient(t *testing.T) {
 	dut := ondatra.DUT(t, "dut")
 	conn := dialConn(t, dut, introspect.GNMI, 9339)
 	c := gpb.NewGNMIClient(conn)
-	if _, err := c.Get(context.Background(), &gpb.GetRequest{Encoding: gpb.Encoding_JSON_IETF, Path: []*gpb.Path{{Elem: []*gpb.PathElem{}}}}); err != nil {
+
+	var req *gpb.GetRequest
+	if deviations.GNMIGetOnRootUnsupported(dut) {
+		req = &gpb.GetRequest{
+			Path: []*gpb.Path{{
+				Elem: []*gpb.PathElem{
+					{Name: "system"}, {Name: "state"}, {Name: "hostname"}}},
```

Comment:
can we add a check for type config and operational as well here if it is supported

Reply:
kindly also add check for type config and operational here as per definition 'https://github.com/openconfig/reference/blob/master/rpc/gnmi/gnmi-specification.md#331-the-getrequest-message'

Reply:
@kjahed - Awaiting your response on this one; We need to know if in default state we have support for "Config", "State" and "Operational".
You have responded for "Config" and "State", what about "Operational".
Can you close on this comment?